src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpCodeVisitor.cs

```diff
@@ -16,6 +15,7 @@ public class CSharpCodeVisitor : CodeVisitor<CSharpCodeWriter>
         private const string ItemParameterName = "item";
         private const string ValueWriterName = "__razor_attribute_value_writer";
         private const string TemplateWriterName = "__razor_template_writer";
+        private const int MaxStringLiteralLength = 1024;
```

Comment:
1024 to correspond with the http contexts response stream [buffer size](https://github.com/aspnet/Mvc/blob/47a3aee2c152b44bb75075e1f7494ba2124c82fa/src/Microsoft.AspNet.Mvc.Core/HttpResponseStreamWriter.cs#L24)